.github/workflows/release-snapshot.yml

```diff
@@ -70,7 +40,7 @@ jobs:
       - name: Deploy artifacts
         id: deploy
         if: github.ref == 'refs/heads/main'
-        run: ./deploy.sh
+        run: ./gradlew release -Prelease.useAutomaticVersion=true --stacktrace --warning-mode all
```

Comment:
we have "org.gradle.warning.mode=all" in the gradle.properties file, do we really need this "--warning-mode all" here?